index.html

```diff
@@ -7,6 +7,6 @@
     <title>Welcome to Potionfy!</title>
 </head>
 <body>
-    <h1>Hello Potionfiers!</h1>
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```

Comment:
Could you remove the 'Enjoy your stay.' string and make 'Welcome to Potionfy' more string red?

Reply:
Sure, can do!